GUIDELINES.md

```diff
@@ -0,0 +1,48 @@
+#General
```

Comment:
Guideline paragraphs need to be separated by a blank line to be formatted properly.